Dockerfile

```
@@ -0,0 +1,57 @@
+COPY pyproject.toml uv.lock ./
+
+# Install uv and sync dependencies
+RUN pip install --no-cache-dir uv && \
+    uv sync --frozen --no-dev
+
+FROM python:3.11-slim
+
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)